src/components/address/Address.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react'
+import Multiaddr from 'multiaddr'
 
 const Address = ({ value }) => {
   if (!value) return null
+
+  const ma = Multiaddr(value)
+  const protos = ma.protoNames().concat('ipfs')
```

Comment:
Heads up: we are switching string representation of peer  multiaddrs from `/ipfs/` to `/p2p/` (https://github.com/libp2p/libp2p/issues/79).

In JS land, the default was switched in [js-multiaddr/releases/tag/v7.0.0](https://github.com/multiformats/js-multiaddr/releases/tag/v7.0.0).

To make this future proof, we should support both:

```suggestion
  const protos = ma.protoNames().concat(['ipfs', 'p2p'])
```